optuna/pruners/hyperband.py

```diff
@@ -63,22 +63,17 @@ class HyperbandPruner(BasePruner):
 
         We minimize an objective function with Hyperband pruning algorithm.
 
-        .. testsetup::
+        .. testcode::
 
-            import numpy as np
+            from sklearn.datasets import load_iris
```

Comment:
Was the `numpy` import intentionally removed? It still seems to pass `doctest` so I guess it's imported in some preamble but it seemed a bit inconsistent to me.

Reply:
You're absolutely right. I happened to remove it during git-rebase. I add `numpy` in https://github.com/optuna/optuna/pull/1221/commits/825a207250dc933e77738b7e07e582ddf81ea140. Thank you for your careful review.